enterprise/frontend/src/metabase-enterprise/content_translation/components/ContentTranslationConfiguration.tsx

```diff
@@ -0,0 +1,158 @@
+import {
+  type ChangeEvent,
+  type ReactNode,
+  useCallback,
+  useRef,
+  useState,
+} from "react";
+import { t } from "ttag";
+
+import { UploadInput } from "metabase/components/upload";
+import ExternalLink from "metabase/core/components/ExternalLink";
+import Markdown from "metabase/core/components/Markdown";
+import {
+  Form,
+  FormProvider,
+  FormSubmitButton,
+  useFormContext,
+} from "metabase/forms";
+import { useSelector } from "metabase/lib/redux";
+import { getDocsUrl } from "metabase/selectors/settings";
+import { Group, Icon, Loader, Stack, Text } from "metabase/ui";
+import { useUploadContentTranslationDictionaryMutation } from "metabase-enterprise/api";
+
+export const ContentTranslationConfiguration = () => {
+  const availableLocalesDocsUrl = useSelector((state) =>
+    // eslint-disable-next-line no-unconditional-metabase-links-render -- This is used in admin settings
+    getDocsUrl(state, {
+      page: "configuring-metabase/localization",
+      anchor: "supported-languages",
+    }),
+  );
+
+  return (
+    <Stack gap="sm" maw="38rem">
+      <Text>
+        <Markdown
+          components={{
+            strong: ({ children }: { children: ReactNode }) => (
+              <ExternalLink href={availableLocalesDocsUrl}>
+                {children}
+              </ExternalLink>
+            ),
+          }}
+        >
+          {t`You can upload a translation dictionary. We'll use this to translate user-provided strings (like question names) into the viewer's language. (Built-in strings won't be affected.) Your translation dictionary should be a CSV with three columns: Locale code, String, Translation. Supported locale codes are **listed here**. Uploading a new dictionary will replace the existing translations.`}
```

Comment:
I think downloading a template CSV here would be very helpful, the listed format `Locale code, String, Translation` differs from the format you gave in the PR. unclear how fussy it is
```
Locale,String,Translation
de,Hello,Guten tag
```

Also probably worth running the copy by @jeff-bruemmer . Could probably be a little more concise? 🪄